docs/act-on-metadata/impact-analysis.md

```diff
@@ -0,0 +1,93 @@
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
+
+# About DataHub Impact Analysis
+
+<FeatureAvailability/>
+
+Impact Analysis is a powerful workflow for understanding the full set of upstream and downstream dependencies of a Dataset, Dashboard, Chart, and many other DataHub Entites.
```

Comment:
It's a bit confusing -- is Impact Analysis a specific concept ? Or a feature?

To me we should just call this "Lineage" to avoid confusion. People will click "Lineage" to use the feature! Specific workflows can be a subset

Reply:
@jjoyce0510 hmmm.. great question. We have a separate Lineage section under "Enrich Metadata", focused on the various ways you can ingest Lineage (Vineetha is on point to build out a better guide, basically building a guide off of our DataHub 101: Lineage video).

This one falls under the "Act on Metadata" section, so I think it still makes sense to call it Impact Analysis. I know we took that terminology out of the UI, but we continue to refer to it as the main lineage-related workflow. My preference would be to keep it as-is for now.

What do you think?

Reply:
imo we should try to get the word lineage into the title, or at least mention that it is powered by lineage in the first sentence

Lineage is our top-searched term by a wide margin (as per algolia stats)

Reply:
@jjoyce0510 just saw @hsheth2's note about high search volume for the term `Lineage`. How about we meet in the middle & I rephrase it as `Lineage Impact Analysis`?